herbert_core/applications/multifit/@mfclass/private/set_fun_private_.m

```diff
@@ -0,0 +1,122 @@
+function [ok, mess, obj] = set_fun_private_ (obj, isfore, args)
```

Comment:
The PR is showing this as a new file. 

There is existing an existing `set_fun_private_(...)` method defined in `mfclass`:  `herbert_core/applications/multifit/@mfclass/set_fun_private_.m` which is amended but not removed on this PR.

Reply:
yes, it should be deleted. It have probably been erroneously saved from Matlab editor